docs/release-notes/mongoid-8.0.txt

```diff
@@ -485,3 +485,21 @@ feature flag turned off, values assigned to fields of type ``BigDecimal`` are
 stored as Strings. See the section on :ref:`BigDecimal Fields <bigdecimal-fields>`
 for more details.
 
+Changes to the ``attributes_before_type_cast`` Hash
+---------------------------------------------------
+
+The ``attributes_before_type_cast`` hash has been changed to function more like
+ActiveRecord:
+
+  - On instantiation of a new model, the ``attributes_before_type_cast`` hash
+    has the same contents as the ``attributes`` hash.
```

Comment:
This is surely not right. What if the value given to constructor in an attribute is uncastable?

Reply:
Good point.

Reply:
done